llama-index-integrations/readers/llama-index-readers-microsoft-sharepoint/llama_index/readers/microsoft_sharepoint/base.py

```diff
@@ -788,10 +800,8 @@ def load_resource(self, resource_id: str, **kwargs) -> List[Document]:
 
             item = self._get_item_from_path(path)
 
-            input_file_dir = path.parent
-
             with tempfile.TemporaryDirectory() as temp_dir:
-                metadata = self._download_file(item, temp_dir, input_file_dir)
```

Comment:
this is another bugfix - we were passing an extra unexpected param to `_download_file`